src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Routes, Route } from 'react-router-dom';
+
+import Home from './components/Home';
+import Login from './components/Login';
+import './App.css';
+import AddRoom from './pages/AddRoom';
+import DeleteRoom from './components/DeleteRoom';
+import Current from './components/Current';
+import Reservations from './components/Reservations';
+import RoomDetails from './components/Detail';
+import SignUp from './components/SignUp';
+import Reservation from './pages/Reservation';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/235856333-643b68d3-df70-4c32-ad82-91c798e73635.png)

That way, your code will comply with the latest standards.  💪